src/triage/component/audition/__init__.py

```diff
@@ -225,6 +239,7 @@ def plot_selection_rules(self):
             self.selection_rule_plotter.plot_all_selection_rules(**common_kwargs)
             self.selection_rule_performance_plotter.plot(plot_type='regret', **common_kwargs)
             self.selection_rule_performance_plotter.plot(plot_type='metric', **common_kwargs)
+            self.results_for_rule = self.selection_rule_performance_plotter.results_for_rule
```

Comment:
How is this used? At first I was very confused because this is the same name as a method on the selection rule picker, but instead seems to refer to a piece of data that it computes. But it happens within the metric loop, so if multiple metrics are present this will have weird behavior. If we want to expose some of this intermediate plot data to users of the Auditioner class, I think it might be more clear to have the Auditioner call `generate_plot_data` directly, save its results, and then pass the results to be plotted.